### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by Michael0x18**
I tried to improve the docs such that they're a little easier for people to understand. Specifically, I added Jacobly's getCSC codes to the keypadc section, and added in most of the remaining notes to graphx (probably saw me asking about whether certain functions changed stuff, whether use cases were safe, etc.)

### CODE REVIEW

**Comment by Michael0x18** [comment]
NOTE: I also changed the makefile a bit to make it build on my system; you can probably ignore that.

File: src/graphx/graphx.h [hunk 1]
```diff
@@ -1009,7 +1023,7 @@ void gfx_PrintUInt(unsigned int n, uint8_t length);
  * Prints a string.
  *
  * Outputs a string at the current cursor position.
- * @param string Pointer to string to print.
+ * @param string Pointer to string to print.n
```
Comment (mateoconlechuga) [inline src/graphx/graphx.h:1026]: look an extra n

File: src/graphx/graphx.h [hunk 2]
```diff
@@ -1063,19 +1081,21 @@ typedef enum {
 
 /**
  * Sets the text foreground color.
+ * Note that if you haven't called
```
Comment (mateoconlechuga) [inline src/graphx/graphx.h:1084]: seems to be missing the end of this statement

File: src/graphx/graphx.h [hunk 3]
```diff
@@ -1063,19 +1081,21 @@ typedef enum {
...
  *
  * @param color Color index to set.
  * @returns Previous text foreground color index.
  * @note Default text foreground color index: 0.
+ * @note Default text background color index is 255, so if you don't change it and try to draw with color 255, nothing will happen.
```
Comment (mateoconlechuga) [inline src/graphx/graphx.h:1089]: perhaps clarification on how to do this properly would be good

File: src/graphx/graphx.h [hunk 4]
```diff
@@ -982,8 +994,9 @@ void gfx_PrintChar(const char c);
 /**
  * Prints a signed integer.
  *
- * Outputs at the current cursor position. Padded with leading zeros if
+ * Outputs at the current cursor position. Pads the integer with leading zeros if
  * necessary to satisfy the specified minimum length.
+ * For example, printInt(5,3) prints "005".
```
Comment (mateoconlechuga) [inline src/graphx/graphx.h:999]: I think you mean "gfx_PrintUInt" here

**Review by mateoconlechuga — CHANGES_REQUESTED** [review]
make the changes in the comments and I'll merge

**Comment by mateoconlechuga** [comment]
Is there a reason misc.rst was added in this PR? there doesn't seem to be anything in it.

**Comment by Michael0x18** [comment]
It's a placeholder: I'm planning to fill it in with little snippets of stuff.
You're right, though. I'll remove it for this merge.

File: src/graphx/graphx.h (hunk 4 above)
Comment (Michael0x18) [reply src/graphx/graphx.h]: Yeah, I changed it to gfx_PrintInt (it's in)

File: src/graphx/graphx.h (hunk 3 above)
Comment (Michael0x18) [reply src/graphx/graphx.h:1088]: Added

File: src/graphx/graphx.h (hunk 2 above)
Comment (Michael0x18) [reply src/graphx/graphx.h]: Yeah, sorry. I moved it to the @note block at the end

**Review by runer112 — APPROVED** [review]

File: docs/static/misc.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
+.. _misc:
+
+Misc. Topics
+============
+
+These topics include small descriptions of how to do some common tasks.
+It's less of a reference for the toolchain and more of a how-to guide.
+
+
```
[the remark this answers is not in the document]
Comment (Michael0x18) [reply docs/static/misc.rst]: I took it out

**Review by mateoconlechuga — APPROVED** [review]

**Pull request merged**